torch/csrc/lazy/ts_backend/README.md

```diff
@@ -0,0 +1,30 @@
+# Lazy Tensor TorchScript Backend
+
+The TorchScript backend runs programs traced via lazy tensor tracing using the torchscript compiler and runtime.  It also serves as a reference backend.
+
```

Comment:
@Krovatkin what level of detail should go here, regarding 'how the TS backend works'?

should i put more of that in the design doc? or more of it here?

Reply:
- give general desc of how IR and 'lowering' works
- point to TS docs (which explain about our executors/TS graph/IValues
- explain why codegen (based on aten schemas) makes sense 
- link to ts_ yaml, generated code infra, 
what else?